src/main/java/org/mskcc/cbio/portal/scripts/ImportProfileData.java

```diff
@@ -131,22 +131,23 @@ public void run() {
                         geneticProfile.getGeneticProfileId(), 
                         genePanel, 
                         geneticProfile.getOtherMetaDataField("generic_entity_meta_properties"),
-                        daoGeneticAlteration
+                        overwriteExisting,
+                        daoGene
                     );
-                    genericAssayProfileImporter.importData(numLines);
+                    genericAssayProfileImporter.importData();
                 }
             } else if(
                 geneticProfile.getGeneticAlterationType() == GeneticAlterationType.COPY_NUMBER_ALTERATION 
-                && DISCRETE_LONG.name().equals(geneticProfile.getDatatype())
```

Comment:
why `geneticProfile.getDatatype()` -> `geneticProfile.getOtherMetaDataField("datatype")` ?

Reply:
`DISCRETE_LONG` ends up in the database as `DISCRETE` at the end.

In the case when the genetic profile has been loaded to the database previously, we reuse the platform and we want to access how the datatype has been defined in the file with `geneticProfile.getOtherMetaDataField("datatype")` instead of how it is stored in the database `geneticProfile.getDatatype()`